pkg/api/componentreadiness/component_report.go

```diff
@@ -1220,7 +1221,6 @@ func triagedIssuesFor(releaseIncidents *resolvedissues.TriagedIncidentsForReleas
 	impactedJobRuns := sets.NewString() // because multiple issues could impact the same job run, be sure to count each job run only once
 	numJobRunsToSuppress := 0
 	for _, triagedIncident := range triagedIncidents {
-		startNumRunsSuppressed := numJobRunsToSuppress
```

Comment:
This was unused.

Reply:
Isn't is used below?  So at the start of the for loop iteration set startNumRunsSuppressed to the current numJobRunsToSuppress, then loop through the job runs for the current triaged incident and if matches are found increment numJobRunsToSuppress

Outside that loop check to see if numJobRunsToSupress is now greater than startNumRunsSupressed and if so append the triagedIncident.  Otherwise don't include it because it isn't relevant and we don't want it in the UI.

What you have changed it to will include all triaged incidents, even ones with job runs that didn't match, any time there is at least one numJobRunsToSuppress

Reply:
Good catch thanks, pushing a restoration momentarily.